src/server/router/index.ts

```diff
@@ -5,12 +5,14 @@ import superjson from 'superjson';
 import { exampleRouter } from './example';
 import { protectedExampleRouter } from './protected-example-router';
 import { electricityRouter } from './electricity.router';
+import { fuelRouter } from './fuel.router';
 
 export const appRouter = createRouter()
   .transformer(superjson)
   .merge('example.', exampleRouter)
   .merge('question.', protectedExampleRouter)
-  .merge('electricity.', electricityRouter);
+  .merge('electricity.', electricityRouter)
+  .merge('fuel', fuelRouter);
```

Comment:
Minor nitpick, you missed the '.' after fuel.

Reply:
Added period, thanks for catching that!